src/Sql/Vault/dbo/Stored Procedures/SecurityTask/SecurityTask_ReadById.sql

```diff
@@ -0,0 +1,10 @@
+CREATE PROCEDURE [dbo].[SecurityTask_ReadById]
+    @Id UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+SELECT *
+FROM [dbo].[SecurityTaskView]
```

Comment:
⛏️ I think there's some formatting weirdness from our norm here, as well as some lower files. I have noticed VS Code at least isn't doing what I'd expect. Running `prettier` on all changed files may get it.

Reply:
Fixed formatting in [8ada50c](https://github.com/bitwarden/server/pull/5025/commits/8ada50cc3b73a03ab7b8fc8c9b0bf3287869c58b)